src/Atlas.cc

```diff
@@ -352,6 +354,17 @@ void Atlas::PostLoad()
     mvpBackupMaps.clear();
 }
 
+void Atlas::SaveAtlas(string pathSaveFileName, string strVocabularyName, string strVocabularyChecksum) {
+    unique_lock<mutex> lock(mMutexAtlas);
+    this->PreSave();
+    cout << pathSaveFileName << endl;
+    std::ofstream ofs(pathSaveFileName, std::ios::binary);
+    boost::archive::binary_oarchive oa(ofs);
+    oa << strVocabularyName;
+    oa << strVocabularyChecksum;
+    oa << this;
```

Comment:
[opt] It would be good to verify that writing out `this` behaves as expected, but we can also check this in combination with John's commit to load any existing map.

Reply:
Added a TODO comment